package/MDAnalysis/visualization/streamlines.py

```diff
@@ -85,7 +89,11 @@ def produce_grid(tuple_of_limits, grid_spacing):
     return grid
 
 
-def split_grid(grid, num_cores):
+# typing: numpy
+def split_grid(
+        grid: np.ndarray,
+        num_cores: int
+) -> Tuple[List[np.ndarray], List[np.ndarray], int, int]:
```

Comment:
```
package/MDAnalysis/visualization/streamlines.py:160: error: Incompatible return value type (got "Tuple[Any, List[List[int]], int, int]", expected "Tuple[List[ndarray], List[ndarray], int, int]")
```

```suggestion
) -> Tuple[List[List[int]], List[List[int]], int, int]:
```

This one surprises me. Also, it may still complain about the first element being `Any` for some reason :thinking: